### PR TITLE
Deck builder deck hover

### PR DESCRIPTION
**Pull request by lily-pytel**
* Pretty much the same as the card hover, except I put the small version here. Is that okay or do we want the full version?
* Also no delay here... seems unnecessary.
* I put the hover effect just on the name cell. It was a bit unseemly to do on the entire row.

Looks like this (but imagine a mouse on the name of the card):
image

@alexmiddeleer @PapaCapricorn

### CODE REVIEW

File: next/components/deck-card-table-row.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  const color = backgroundColor ? theme.cardTableName : 'white';
+  const imagePath = imagePathSmall(card.name, card.set || undefined);
+  const smallImageWidth = IMAGE_SIZES.smallImageWidthPortrait;
+
+  return (
+    <tr key={card.id} data-cy="deckCardRow">
+      <style jsx>{`
+        td {
+          padding: 2px 5px 5px 5px;
+          border: 1px solid #03080a;
```
Comment (alexmiddeleer) [inline next/components/deck-card-table-row.js]: Using this in 2 spots now. Time for a color variable? "border-light" perhaps? I know it's dark, but we have a dark theme so it's basically a light border.

File: next/components/deck-card-table-row.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  const backgroundColor = cardMainColor(card, theme);
+  const color = backgroundColor ? theme.cardTableName : 'white';
+  const imagePath = imagePathSmall(card.name, card.set || undefined);
+  const smallImageWidth = IMAGE_SIZES.smallImageWidthPortrait;
+
+  return (
+    <tr key={card.id} data-cy="deckCardRow">
+      <style jsx>{`
+        td {
+          padding: 2px 5px 5px 5px;
```
Comment (alexmiddeleer) [inline next/components/deck-card-table-row.js]: could do `2px 5px 5px` Not even sure I prefer that notation, this is more of an FYI.

File: next/components/deck-card-table-row.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+        @media (hover: hover) {
+          // Show the hover image (but only on devices that have hover)
+          .imgWrapper:hover::before {
+            visibility: visible;
+            opacity: 1;
+          }
+        }
+      `}</style>
+      <td>{card.mana < 0 ? 'X' : card.mana}</td>
```
Comment (alexmiddeleer) [inline next/components/deck-card-table-row.js]: we might want to move this to a formatter function in card utils. It's used in 2 spots

File: next/components/deck-card-table-row.js (hunk 2 above)
Comment (lily-pytel) [reply next/components/deck-card-table-row.js]: I am not a fan of that shortened notation, personally

File: next/components/deck-card-table-row.js (hunk 1 above)
Comment (lily-pytel) [reply next/components/deck-card-table-row.js]: Done. Let me know if the variable is too specific. We already have other borders so I prefer to give them more specific names (for the theme-context, of course, not the color itself, which is just a super dark gray) @alexmiddeleer

File: next/components/deck-card-table-row.js (hunk 3 above)
Comment (lily-pytel) [reply next/components/deck-card-table-row.js]: Done

**Comment by lily-pytel** [comment]
@alexmiddeleer good point, I changed it so the image appears when you hover everywhere on the name line

**Comment by lily-pytel** [comment]
Fixes #182

**Pull request merged**